.github/workflows/website.yaml

```
@@ -1,6 +1,10 @@
 name: 'Website update'
 on:
   workflow_dispatch:
+    inputs:
+        kimai_version:
+            description: 'The Kimai version to update the website for'
+            required: true
```

Indendation looks off, the rest of the file uses 2 spaces...
```suggestion
      kimai_version:
          description: 'The Kimai version to update the website for'
          required: true
```

---

`type: string` is default?